docs/docs/guides/basics/eth.md

```diff
@@ -344,6 +343,838 @@ Transaction receipt: {
+
+The [getBlock](/api/web3-eth/function/getBlock) function typically takes two parameters:
+- block ([HexString32Bytes](/api/web3/namespace/types/#HexString32Bytes) | [BlockNumberOrTag](/api/web3/namespace/types/#BlockNumberOrTag)) - Number of block or tag
+- hydrated (boolean) -  if true, it returns the detail of each transaction. If false, only the hashes of the transactions.
+
+The function returns a Promise that resolves to an object containing details about the requested block. The structure of this object may vary depending on whether you requested transaction objects. The block information often includes fields like number, hash, parentHash, nonce, miner, gasLimit, gasUsed, timestamp, and more.
+```typescript
+import Web3 from 'web3';
+const web3 = new Web3('http://localhost:7545');
+
+async function test() {
+    const block = await web3.eth.getBlock('latest');
+    console.log('Block:', block);
+    // Block: {
+    //     number: 0n,
+    //     hash: '0x1218eb233187b0120a09fea9eae5d5360df5abbad66cb59d0f93538fcf4748e5',
+    //     parentHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
+    //     mixHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
+    //     nonce: 0n,
+    //     sha3Uncles: '0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347',
+    //     logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+    //     transactionsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
+    //     stateRoot: '0xf1613937d7faa51ee731f0f2c7a7828ab0ddd2e4644a268d3be2f1b315b023d1',
+    //     receiptsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
+    //     miner: '0x0000000000000000000000000000000000000000',
+    //     difficulty: 0n,
+    //     totalDifficulty: 0n,
+    //     extraData: '0x',
+    //     size: 1000n,
+    //     gasLimit: 6721975n,
+    //     gasUsed: 0n,
+    //     timestamp: 1698721176n,
+    //     uncles: []
+    // }
+}
+(async () => {
+    await test();
+})();
+```
+
+### getBlockNumber
+
+The [getBlockNumber](/api/web3-eth/function/getBlockNumber) function is used to retrieve the latest block number (also known as the block height) of the Ethereum blockchain. The block number is a crucial piece of information in Ethereum as it represents the current state of the blockchain, indicating the total number of blocks that have been mined up to the present.
```

Comment:
Good effort, I think just making some changes as mentioned above will make it for 'how to get most out of web3' / web3 cookbook / tutorials